### PR TITLE
dependencies

### PR DESCRIPTION
**Pull request by Mouvedia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ The special type `Any` means that any type is allowed:
 
 #### The `Void` Type
 
-The special type `Void` should only be used to indicate that a function doesn’t return any value.
+The special type `Void` should only be used to indicate that a function doesn’t return any value or returns `undefined`.
```
Comment (ericelliott) [inline README.md]: "Doesn't return any value" and "returns undefined" are the same thing.

File: README.md [hunk 2]
```diff
@@ -181,6 +181,20 @@ functionName() => String, throws: Error
 ```
 
 
+### Dependencies
+
+If necessary you can list the non built-in functions required, which is especially useful if you are using polyfills.
```
Comment (ericelliott) [inline README.md]: > "If necessary you can list the non built-in functions required"

Would this be easier to understand?

"You can optionally list your function's dependencies. In the future, add on tools may automatically scan your functions and list dependencies for you, which could be useful for documentation and to identify polyfills required for your application to run on browsers and platforms that lack the required features."

File: README.md [hunk 3]
```diff
@@ -181,6 +181,20 @@ functionName() => String, throws: Error
...
+
+```js
+// one dependency
+functionA() => Type, requires: functionB
+
+// several dependencies
+functionA()
+  => Type,
+  requires: [functionB, functionC]
```
Comment (ericelliott) [inline README.md]: Brackets should not be needed.

**Comment by tomek-he-him** [comment]
:+1: :+1:

**Pull request merged**